### ISSUE TITLE
Design user flow

### ISSUE THREAD

**Issue by mattgawarecki** [EssDeeDee/safe-eats-ui#1]

**Comment by astrocaribe** [comment]
*User Experience Workflows*
Search by establishment name
Search by establishment location
View establishment
- Name
- Location
- Most recent score (edited)

**Comment by astrocaribe** [comment]
Trying to capture what we decided earlier. Is that where this^^^ goes?

**Comment by mattgawarecki** [comment]
Yeah, that's actually a great place for it. Thanks for the assist!
